.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: Main
+
+on:
+  push:
+    branches: [ main, release/*]
+  pull_request:
+
+jobs:
+  Build:
+    runs-on: macos-11
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["iOS", "tvOS"]
+        include:
+            - name: "iOS"
+              scheme: "Container"
+              destination: "platform=iOS Simulator,OS=15.0,name=iPhone 12 Pro"
+              test: true
+              lint: true
+
+            - name: "tvOS"
+              scheme: "Container"
+              destination: "platform=tvOS Simulator,OS=15.0,name=Apple TV 4K"
+              test: true
+              lint: true
```

Comment:
Do both the iOS and tvOS matrix entries need to run `pod lib lint`?

Reply:
Nope, funny you should say that. I'm in the process of finishing up a "project restructure" for all the projects that addresses this as well - it splits out linting for CP (and SPM) into another job.

https://github.com/BottleRocketStudios/iOS-Container/pull/13